manifests/d/Dylibso/Extism/CLI/1.6.1/Dylibso.Extism.CLI.locale.en-US.yaml

```diff
@@ -0,0 +1,27 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Dylibso.Extism.CLI
+PackageVersion: 1.6.1
+PackageLocale: en-US
+Publisher: Dylibso
+PublisherUrl: https://dylib.so/
+PublisherSupportUrl: https://github.com/extism#support
+PackageName: Extism CLI
+PackageUrl: https://extism.org/
+License: BSD-3-Clause License
+LicenseUrl: https://github.com/extism/cli/blob/main/LICENSE
+ShortDescription: Extism CLI generates and executes Extism plugins, and manages libextism installations.
+Tags:
+- wasm
+- wasi
+ReleaseNotes: |
+  Commits
+  - feat: support enable-http-response-headers flag, minor tweaks
+Icons:
+- IconUrl: https://avatars.githubusercontent.com/u/84687501
+  IconFileType: png
+  IconResolution: 256x256
+  IconTheme: default

```

Comment:
```
Manifest Warning: Field usage requires verified publishers. [Icons]
```

Try removing [Icons] from the manifest.

Reply:
Icons are not accepted in the community repository. This should be removed from the manifest